modules/cookbook/pages/recipe-configmgmt.adoc

```diff
@@ -0,0 +1,113 @@
+[[cookbook-config-mgmt]]
+= Configuration Management
+
+You can use configuration files and channels to manage configuration for your clients, rather than configuring each client manually.
+This recipe shows you how to use the {productname} {webui} to create a centrally-managed configuration file, assign it to a Salt client, and apply the configuration.
+
+
+
+== Problem
+
+If you are managing a lot of clients, you probably don't want to manually apply configuration settings to each of them in turn.
+Configuration channels are used to organize configuration files.
+You can subscribe clients to configuration channels, and deploy configuration files as required.
+
+
+
+== Solution
+
+When you have completed this recipe, you will have a configuration channel containing a configuration file, have assigned clients to the channel, and applied the configuration successfully.
+
+
+
+== Before you begin
+
+Before you start, you should already have:
+
+* A {productname} Server installed, that you can access using the {webui}.
+* At least one Salt client registered to your server.
+* Appropriate subscriptions from http://scc.suse.com for the products you are using.
```

Comment:
I think this is optional (Uyuni).

Reply:
True, although there's a lot of caveats in that sentence ("appropriate" etc)